linode/manager
# upcoming: [DI-22596] - Changes for adding overview section in the alert details page in Cloud Pulse.

## Description 📝

Changes for adding overview section in the alert details page in Cloud Pulse.

## Changes  🔄

1. Add new AlertDetailOverview component and its respective changes.
2. Add new AlertDetailRow component that will be used in different sections in alert details page

## Target release date 🗓️
07-01-2024

## Preview 📷

**Include a screenshot or screen recording of the change.**

:lock: Use the [Mask Sensitive Data](https://cloud.linode.com/profile/settings) setting for security.

:bulb: Use `<video src="" />` tag when including recordings in table.

| Before  | After   |
| ------- | ------- |
|![Screenshot 2024-12-31 at 3 32 17 PM](https://github.com/user-attachments/assets/10694c5d-1775-4c14-b08b-a962d09657fd)|![Screenshot 2024-12-31 at 3 32 36 PM](https://github.com/user-attachments/assets/4037fbff-9b72-42d7-96dc-65e86de1ced8)|
|![Screenshot 2024-12-31 at 3 33 49 PM](https://github.com/user-attachments/assets/f325676b-16b0-4f48-a8b2-fd31b1e19dab)|![Screenshot 2024-12-31 at 3 34 35 PM](https://github.com/user-attachments/assets/f85f87f0-b66f-40a5-857f-daea96c04c48)|
|![Screenshot 2024-12-31 at 3 35 24 PM](https://github.com/user-attachments/assets/15c79e55-79a2-4f8f-b665-06d07bac389a)||

## How to test 🧪

1. Login as legacy mock user as some endpoints related to alerting not available on production
2. Navigate to monitor tab.
3. Navigate to alerts sub tab.
4. On any alerts click on show details action menu.
5. You should be able to see the details page with overview section

Figma - https://www.figma.com/design/DoGpj6YJ8cHGqSHQIcSTrI/System-Alerts?node-id=1609-109092&t=C78AUJaAMFpLht1u-0

<details>
<summary> Author Checklists </summary>

## As an Author, to speed up the review process, I considered 🤔

👀 Doing a self review
❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
🤏 Splitting feature into small PRs
➕ Adding a [changeset](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md#writing-a-changeset)
🧪 Providing/improving test coverage
 🔐 Removing all sensitive information from the code and PR description
🚩 Using a feature flag to protect the release
👣 Providing comprehensive reproduction steps
📑 Providing or updating our documentation
🕛 Scheduling a pair reviewing session
📱 Providing mobile support
♿  Providing accessibility support

<br/>


## As an Author, before moving this PR from Draft to Open, I confirmed ✅


</details>
